apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/internal/SimpleResponseReader.kt

```diff
@@ -125,12 +124,12 @@ class SimpleResponseReader private constructor(
         val conditionValue = variableValues[condition.variableName] as Boolean
         if (condition.inverted) {
           // means it's a skip directive
-          if (java.lang.Boolean.TRUE == conditionValue) {
```

Comment:
I want to say that this is the same as before. But I'm not sure.

Reply:
Yeah, that's fine. It was bad port to kotlin.